gen-certs.sh

```diff
@@ -9,11 +9,16 @@ CITY=${CITY:-Nuremberg}
 STATE=${STATE:-Bavaria}
 COUNTRY=${COUNTRY:-DE}
 
-DIR="/etc/pki"
+DIR="${DIR:-/etc/pki}"
 CERTS="$DIR/_certs"
 PRIVATEDIR="$DIR/private"
 WORK="$DIR/_work"
 
+
+random_serial() {
+    python -c 'import random ; print(random.randint(1, 2**159))'
```

Comment:
What is the max acceptable serial number? If it's 2**159 - then, if we choose that (or anything close to it..) we'll be in trouble. We'll need to ensure we have enough room for ever cert + renewal we'll ever issue!